src/scss/components/utility/_utility.scss

```diff
@@ -2,17 +2,50 @@
 
 // Utility Link on Hero banner
 
+// Utility button.
+.config_pages--view-mode--site-settings-header {
+  a {
+    font-family: $su-font-sans;
+    font-weight: 500;
+
+    &.su-button {
+      background: $vpue-gradient-sea-to-light-sea-utility;
+
+      &:hover,
+      &:focus {
+        color: $su-color-white;
+        background: $su-color-bright-red;
+      }
+    }
+  }
+}
+
+// Utility button overrides needed.
+.su-mobile-utility-button {
+  a {
+    &.su-button {
+      background: $vpue-gradient-sea-to-light-sea-utility;
+
+      &:hover,
+      &:focus {
+        color: $su-color-white;
+        background: $su-color-bright-red;
+      }
+    }
+  }
+}
+
+// Homepage changes.
```

Comment:
@jenbreese It looks like this was merged, but I'd recommend combining these. Like this:


// Utility button overrides needed.
.su-mobile-utility-button,
.config_pages--view-mode--site-settings-header {
  a {
    &.su-button {
      background: $vpue-gradient-sea-to-light-sea-utility;

      &:hover,
      &:focus {
        color: $su-color-white;
        background: $su-color-bright-red;
      }
    }
  }
}`